lib/backend/k8s/k8s.go

```diff
@@ -93,27 +191,41 @@ func (c *KubeClient) Syncer(callbacks api.SyncerCallbacks) api.Syncer {
 
 // Create an entry in the datastore.  This errors if the entry already exists.
 func (c *KubeClient) Create(d *model.KVPair) (*model.KVPair, error) {
-	log.Warn("Attempt to 'Create' using kubernetes backend is not supported.")
-	return nil, errors.ErrorResourceDoesNotExist{
-		Err:        goerrors.New("Resource does not exist"),
-		Identifier: d.Key,
+	switch d.Key.(type) {
+	case model.GlobalConfigKey:
+		return c.createGlobalConfig(d)
+	default:
+		log.Warn("Attempt to 'Create' using kubernetes backend is not supported.")
+		return nil, errors.ErrorResourceDoesNotExist{
```

Comment:
Feels like we should have an ErrorActionNotSupported   or some such thing rather than a resource does not exist error.